addons/common/functions/fnc_isRemoteControlled.sqf

```diff
@@ -0,0 +1,22 @@
+#include "script_component.hpp"
+/*
+ * Author: NeilZar
+ * Check if the given unit is currently remote controlled.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Unit is Remote Controlled <BOOL>
+ *
+ * Example:
+ * [_unit] call zen_common_fnc_isRemoteControlled
+ *
+ * Public: Yes
+ */
+
+params ["_unit"];
+
+_curators = allCurators apply {getAssignedCuratorUnit _x} select {!(isNull _x)};
+
+_unit getVariable ["bis_fnc_moduleremotecontrol_owner", objNull] in _curators
```

Comment:
Missing private but, I think we could use `allPlayers` command here instead.

Reply:
I don't think using `allPlayers` is appropriate here. That command could return way more than `allCurators`, my community has 50 player missions regularly and only 4 Zeus slots. I also don't see how `allPlayers` would improve it because, as far as I can see, there is no reason any other than a Zeus would open an arsenal on a remote-controlled unit.